Source/Scene/Model.js

```diff
@@ -564,6 +567,18 @@ define([
         },
 
         /**
+         * Gets a promise that resolves when the model is ready to render.
```

Comment:
Can we make this doc as good as the doc for `this.readyToRender` and then change all doc links to `readyToRender` to this?